src/main/java/games/strategy/engine/persistence/serializable/ProxyFactoryRegistryFactory.java

```diff
@@ -0,0 +1,24 @@
+package games.strategy.engine.persistence.serializable;
+
+import games.strategy.internal.persistence.serializable.PropertyBagMementoProxy;
+import games.strategy.internal.persistence.serializable.VersionProxy;
+import games.strategy.persistence.serializable.ProxyFactoryRegistry;
+
+/**
+ * Factory for creating pre-configured instances of {@link ProxyFactoryRegistry}.
+ */
+public final class ProxyFactoryRegistryFactory {
+  private ProxyFactoryRegistryFactory() {}
+
+  /**
+   * Creates a new proxy factory registry that has been configured with all platform proxy factories.
+   *
+   * @return A new proxy factory registry; never {@code null}.
+   */
+  public static ProxyFactoryRegistry newPlatformProxyFactoryRegistry() {
+    final ProxyFactoryRegistry proxyFactoryRegistry = ProxyFactoryRegistry.newInstance();
+    proxyFactoryRegistry.registerProxyFactory(PropertyBagMementoProxy.FACTORY);
```

Comment:
nit: this list will grow, may be good to start using a more minimal syntax now since lines will likely be added via copy paste. EG:
```
Arrays.asList(
  PropertyBagMementProxy.FACTORY,
  VersionProxy.FACTORY
).forEach(proxyFactoryRegistry::regsitryProxyFactory);
```